modules/game_module.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../classes/game'
+require 'json'
+
+module GamesDataController
+  def load_games
+    file = './json/games.json'
+    data = []
+    if File.exist?(file) && File.read(file) != ''
+      JSON.parse(File.read(file)).each do |element|
+        data.push(Game.new(element['multiplayer'], element['last_played_at'], element['published_date']))
+      end
+    end
+    data
+  end
+
+  def save_games
+    data = []
+    puts @games.to_s
+    @games.each do |game|
+      data.push({ multiplayer: game.multiplayer, last_played_at: game.last_played_at,
+                  published_date: game.publish_date.to_s })
+    end
+    File.write('./json/games.json', JSON.generate(data))
```

Comment:
- [ ] [REQUIRED] There is no json folder in your codebase but you are specifying './json' to save your game data, so your code will name be able to save. Kindly fix this please

Reply:
Fixed

Reply:
Kindly note: that I am still working on this PR. I will request for a review once done.